### DISCUSSION TITLE
Welcome to FreeRDP Discussions!

### DISCUSSION THREAD

**Discussion by bmiklautz** [FreeRDP/FreeRDP#6652] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by srivatsagowda** [comment]
how to enable rails mode in freerdp?

**Reply by akallabeth** [reply]
1. You need to have RAILS/Remote App enabled on your server 
1. Client side add `/app:"someapp.exe"` to your command line or use a generated `RDP` file (`webaccess`, ...)

**Reply by srivatsagowda** [reply]
Thank you .
By the way I'm new to freerdp. where can I find resources to understand the flow of  freerdp source code apart from github documentation ?

**Reply by akallabeth** [reply]
See https://www.freerdp.com/ and the linked wiki/...

**Reply by srivatsagowda** [reply]
Actually, I wanted to get the title of the window opened in the server using freerdp. Is there a way where  we can get the title without enabling RAILS?

**Reply by akallabeth** [reply]
in short:
1. if you need a desktop session: screen scraping / some custom channel extension that needs to be installed on client and server or no chance
2.  `RAILS / Remote APP`: modify the client to dump out the window data it creates (maybe already in the debug log)

**Reply by srivatsagowda** [reply]
Thank You. helped a lot.

**Reply by srivatsagowda** [reply]
Hi 
I'm trying to create static virtual channel to get the window title data from server using a script running at server side. How to build dynamic virtual channel in freerdp?
